public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -105,60 +142,27 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
   /**
    * Returns the body (layout) or the full view panel
    */
-  getBodyToRender(viewPanelKey?: string): SceneObject {
+  public getBodyToRender(viewPanelKey?: string): SceneObject {
     const viewPanel = findVizPanel(this, viewPanelKey);
     return viewPanel ?? this.state.body;
   }
-}
-
-class DashboardSceneUrlSync implements SceneObjectUrlSyncHandler {
-  constructor(private _scene: DashboardScene) {}
 
-  getKeys(): string[] {
-    return ['inspect', 'viewPanel'];
+  private startTrackingChanges() {
+    this._changeTrackerSub = this.subscribeToEvent(
```

Comment:
This is quite elegant I must say, comapred to what we currently have in dashboards :)